### PR TITLE
AB#  Added spinner web component

### PR DESCRIPTION
**Pull request by kjec2f**
**Describe pull-request**  
_Describe what the pull-request is about_
A spinner web component has added.
Added a web component to the demo/angular and demo/react folders.
**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: #

**How to test**  
_Add description how to test if possible_
1. Go to... component/spinner
2. Check in...
3. Run ...

**Screenshots**  
_If applicable, add screenshots to help explain_

**Additional context**  
_Add any other context about the pull-request here._

### CODE REVIEW

File: components/src/components/spinner/spinner.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  export class Spinner{
+    @Prop() size = "";
+    @Prop() type = "";
+
+    @State() cubes = 9;
+    @State() rects = 5;
+
+    renderElements(num, className){
+      const elems = [];
```
Comment (helloimela) [inline components/src/components/spinner/spinner.tsx:17]: "elems" value has changed inside for loop. So let's use "let elems" instead of "const elems".

**Comment by helloimela** [comment]
Merge master into the branch & fix conflict

File: theme/light/src/components/_spinner.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+:root,
```
Comment (helloimela) [inline theme/light/src/components/_spinner.scss:1]: You need to import this file into sdds/theme/light/src/theme/sdds-theme.scss
@import '../components/spinner';

**Review by helloimela — CHANGES_REQUESTED** [review]

File: theme/light/src/components/_spinner.scss (hunk 2 above)
Comment (kjec2f) [reply theme/light/src/components/_spinner.scss:1]: Imported

File: components/src/components/spinner/spinner.stories.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  case 'medium':
+    sizeValue = 'md';
+    break;
+  default: sizeValue= '';
+    break;
+}
+
+  return `
+  ${style}
+  <div class="demo">
```
Comment (helloimela) [inline components/src/components/spinner/spinner.stories.js:54]: Need to add <sdds-theme></sdds-theme> in the stories to make it work in storybok preview

**Review by helloimela — CHANGES_REQUESTED** [review]

File: components/src/components/spinner/spinner.stories.js (hunk 3 above)
Comment (kjec2f) [reply components/src/components/spinner/spinner.stories.js:55]: Added.

**Review by helloimela — APPROVED** [review]

**Pull request merged**